### DISCUSSION TITLE
I'm curious why it renders normally in the mujoco simulator but not in Unity.

### DISCUSSION THREAD

**Discussion by mg638** [google-deepmind/mujoco#2143] [Asking for Help]
### Intro

hello. I am working on a Unity project using Mujoco.

I'm asking for help because I think objects that are displayed normally in the Mujoco simulator are not created properly in Unity.

### My setup

Mujoco 3.2.0 Unity Plug-in

### My question

![스크린샷 2024-10-16 182437](https://github.com/user-attachments/assets/7f3a8b7a-ecf4-4524-ae31-aed45b1264f4)
![스크린샷 2024-10-16 182452](https://github.com/user-attachments/assets/04cf5639-1202-440d-ab07-65627dcee209)

The one where the slider is created properly is the Unity simulator, and the one where the slider is not created is the Unity image.

### Minimal model and/or code that explain my question

<tendon>
      <spatial name="slider1" limited="true" range="0 .51" rgba="0 .1 1 1" width=".005">
            <site site="slider1_origin"/>
            <site site="slider1_insertion"/>
      </spatial>
      <spatial name="slider2" limited="true" range="0 0.6" rgba="0 .1 1 1" width=".005">
            <site site="slider2_origin"/>
			<site site="slider2_insertion_temp"/>
            <site site="slider2_insertion"/>
      </spatial>
      <spatial name="slider3" limited="true" range="0 .51" rgba="0 .1 1 1" width=".005">
            <site site="slider3_origin"/>
            <site site="slider3_insertion"/>
      </spatial>
      <spatial name="slider4" limited="true" range="0 0.6" rgba="0 .1 1 1" width=".005">
            <site site="slider4_origin"/>
			<site site="slider4_insertion_temp"/>
            <site site="slider4_insertion"/>
      </spatial>
  ....
</tendon>

XML is produced this way. 
I tried adding a type and tried various things, but it doesn't work.
Can you help me figure out what is causing it?

### Confirmations

- [X] I searched the [latest documentation](https://mujoco.readthedocs.io/en/latest/overview.html) thoroughly before posting.
- [X] I searched previous [Issues](https://github.com/google-deepmind/mujoco/issues) and [Discussions](https://github.com/google-deepmind/mujoco/discussions), I am certain this has not been raised before.

**Comment by yuvaltassa** [comment]
@Balint-H, any thoughts?

**Reply by Balint-H** [reply]
Sorry for not responding, have been busy, but will start to be available again now. Will take a look tomorrow.

**Comment by Balint-H** [comment]
Hello you were correct to submit it as an issue as well (#2165). This is related to the lack of additional MuJoCo plugin support, which I'll try to address by the end of November. Let's use the issue to track this.
